src/controllers/clientController.js

```diff
@@ -0,0 +1,124 @@
+const Client = require('../models/client');
+const Dish = require('../models/dish');
+
+exports.createClient = async (req, res) => {
+  try {
+    const { user, address, contactNumber } = req.body;
+
+    const client = new Client({
+      user,
+      address,
+      contactNumber,
+    });
+
+    const createdClient = await client.save();
+
+    res.status(201).json(createdClient);
+  } catch (error) {
+    res.status(500).json({ error: 'Server error' });
+  }
+};
+
```

Comment:
What will happen if the user tries to create two clients?
We need to check somehow if the client is in the DB or not, then if it's not in the DB then we create a new client.

Reply:
yes that makes sense